backend/app/models.py

```diff
@@ -142,3 +150,13 @@ class Image(db.Model):
 
     def __repr__(self):
         return '<Image %r>' % self.name
+
+
+class RevokedTokenModel(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    jti = db.Column(db.String(120))
+
+    @classmethod
+    def is_jti_blacklisted(cls, jti):
+        query = cls.query.filter_by(jti=jti).first()
+        return bool(query)
```

Comment:
Looks like you only want to determine if a given JWT matching some conditions exists, so why you order DB to download this JWT? :)

Use `.exists()` method, I haven't checked if there is such a method, but I believe there is.